src/Lumi/Components/DropdownButton.purs

```diff
@@ -232,6 +235,38 @@ dropdownMenuDefaults =
   , items: []
   }
 
+type DropdownIconProps =
+ { icon :: JSX
+ , content :: JSX
+ , onOpen :: Effect Unit
+ , alignment :: Nullable String
+ }
+
+dropdownIconDefaults :: DropdownIconProps
+dropdownIconDefaults =
+  { icon: Text.text
```

Comment:
Is there a particular reason you decided not to use the `Overflow` icon here?

Reply:
Nope, I missed it, good catch